src/App.css

```diff
@@ -1,38 +1,47 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
 .App {
   text-align: center;
+  display: flex;
+  flex-direction: column;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.app__header {
```

Comment:
Let's be consistent with the naming. If the BLOCK is called `App` the element header should be `App__header`

Reply:
Same for the rest: `app__nav` etc